zstash/globus.py

```diff
@@ -216,21 +218,37 @@ def globus_transfer(
     try:
         if task_id:
             task = transfer_client.get_task(task_id)
+            # one  of {ACTIVE, SUCCEEDED, FAILED, CANCELED, PENDING, INACTIVE}
             if task["status"] == "ACTIVE":
-                return
+                logger.info(f"{ts_utc()}: Globus task_id {task_id} Still Active")
+                return "ACTIVE"
             elif task["status"] == "SUCCEEDED":
+                logger.info(f"{ts_utc()}: Globus task_id {task_id} status = SUCCEEDED")
                 src_ep = task["source_endpoint_id"]
                 dst_ep = task["destination_endpoint_id"]
                 label = task["label"]
+                ts = ts_utc()
                 logger.info(
-                    "Globus transfer {}, from {} to {}: {} succeeded".format(
-                        task_id, src_ep, dst_ep, label
+                    "{}:Globus transfer {}, from {} to {}: {} succeeded".format(
+                        ts, task_id, src_ep, dst_ep, label
                     )
                 )
             else:
-                logger.error("Transfer FAILED")
+                logger.error(f"{ts_utc()}: Transfer FAILED (task_id = {task_id})")
```

Comment:
I would consider updating this `error` logging to include what the actual `task["status"]` was, because it's not necessarily `"FAILED"`

Reply:
Agreed!  Will modify to issue status.
NOTE: An issue with this entire code-block is that we may or may-not return, may or may-not retain the previous task_id, etc.  Logging needs to clarify which is which.